zcash_proofs/src/circuit/sapling.rs

```diff
@@ -260,6 +279,7 @@ impl<'a, E: JubjubEngine> Circuit<E> for Spend<'a, E> {
 
         assert_eq!(
             note_contents.len(),
+            256 + // asset_type
```

Comment:
where did `256` come from? is that the result of converting the asset type to some bit representation?

Reply:
the specific data in the note is a group element, which is ~256 bits, which is the value base for that asset type. will add to comment

Reply:
Also oops. its the u32 arbitrary asset type, in the note.